### DISCUSSION TITLE
RAG is malfunctioning with the new Ollama versions later than 2.0

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#3907] [General]
# Bug Report

## Description
"The RAG is malfunctioning with the new Ollama versions later than 2.0! Does someone have additional information on the subject?"

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version:** latest
- **Ollama (if applicable):** after 2.0

- **Operating System: Windows 11
- **Browser (if applicable):** Edge and chrome

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
@silentoplayz do we have an issue ticket from Ollama repo?

**Reply by silentoplayz** [reply]
Negative. We do not have a issue open on the Ollama repo.

**Reply by silentoplayz** [reply]
Issue created: https://github.com/ollama/ollama/issues/5736

**Comment by Qualzz** [comment]
I confirm extremely weird behavior with the RAG since the ollama last update. 

![image](https://github.com/user-attachments/assets/3e8861c8-adba-4f74-9d01-5861ebff595d)

The document looks like this: 
![image](https://github.com/user-attachments/assets/33f81ac0-ed82-46ee-a012-3c78dee86351)

Previously it was working extremely fine. Now it's just weird ( for the exact same document/prompt). Not that it can't look in the document. It just doesn't care what's inside. It's hard to explain. 


Another exemple: 

Previously:
![image](https://github.com/user-attachments/assets/2aecf417-60c0-4b66-bda8-c6da4659be57)

Now: 
![image](https://github.com/user-attachments/assets/3d6d0b17-953c-44b4-a14d-460bb6e9cc09)

**Comment by silentoplayz** [comment]
@zouzeTG @Qualzz Please provide me with your CPU and GPU (if applicable) types so that I can submit a sufficient bug report to the Ollama repo.

Which GPU are you using?
```
Nvidia
AMD
Intel
Apple
Other
```

Which CPU are you using?
```
Intel
AMD
Apple
Other
```

Anymore information that either of you could provide for this bug report would be great as well. Thanks in advance!

Edit: Please provide such information on this open issue instead: https://github.com/ollama/ollama/issues/5736

**Reply by zouzeTG** [reply]
Hi @silentoplayz thank's for your action !
I use GPU NVIDIA A100 and Intel CPU with Windows server 2022

**Reply by zouzeTG** [reply]
I'm going to do more test, because for me I have also encontered bug wtih the version 2.1. with docx and xls file
